### DISCUSSION TITLE
Subtitles no longer displayed

### DISCUSSION THREAD

**Discussion by KaiFrederking** [smplayer-dev/smplayer#1016] [Q&A] [answered]
Since the latest update, subtitles are no longer displayed, even in videos where they worked before. VLC displays them without problems. This is true for .srt subtitles, as well as those in the video container (.mkv).
I've gone through all settings, but can't get them to work again. And yes, I did try pressing "V", too ;)
P.S.: Reinstalling the latest version also didn't help.
P.P.S.: OS is Win 10

**Comment by smplayer-dev** [comment]
Be sure the option "Select first available subtitle" in preferences -> subtitles is enabled.

**Reply by KaiFrederking** [reply]
Thank you! Unfortunately, that's already set correctly.
![image](https://github.com/user-attachments/assets/85538a63-afa9-455a-b647-a362c2f79b09)

**Comment by smplayer-dev** [comment]
I can't reproduce this problem in my computer.
Please send the smplayer log after trying to play a video with subtitles.

**Reply by KaiFrederking** [reply]
Thank you for your reply! Here you go - an attempt to play "The Matrix" with .srt subtitles. No error messages, but also no subtitles.
[SMPlayer.log](https://github.com/user-attachments/files/16380543/SMPlayer.log)

**Comment by smplayer-dev** [comment]
Is the subtitle available in the subtitles menu? Can you select it?

**Reply by KaiFrederking** [reply]
Yes, it is and I did:
![image](https://github.com/user-attachments/assets/d0e35cf9-ecd9-4408-ac7d-fbc9d5e5424c)

**Reply by KaiFrederking** [reply]
And since vlc displays them, I assume it's not a problem with the files. 
![image](https://github.com/user-attachments/assets/6f9950ab-e84b-4d5b-9d1d-454d9ddf409a)

**Reply by KaiFrederking** [reply]
P.S.: On selecting the subtitle file, it's even shown as active ... 
![image](https://github.com/user-attachments/assets/d9c71b06-b872-408d-9cca-8219bd2e43ec)

**Reply by KaiFrederking** [reply]
P.P.S.: I also tried reloading the .srt and switching between the two (identical) tracks, but also no effect.

**Comment by smplayer-dev** [comment]
Could you send the subtitle file?

**Reply by KaiFrederking** [reply]
Sure, of course - and again: Thank you for your help.
Still, it's not this movie alone, but ALL formerly working subtitles for all films I've tried so far that aren't working.
Still, 'ere we go (I zipped it since upload here doesn't support .srt)
[The Matrix [1999].zip](https://github.com/user-attachments/files/16400573/The.Matrix.1999.zip)
:

**Comment by smplayer-dev** [comment]
Your subtitles works fine for me.

Maybe the problem is caused by this error in your log, it seems it can't find the font.

```
[17:46:20:697] MplayerProcess::parseLine: '[ass] fontselect: Using default font: (Sans, 400, 0) -> Arial, 0, (none)'
[17:46:20:697] MplayerProcess::parseLine: '[ass] fontselect: (Sans, 400, 0) -> Arial, 0, (none)'
[17:46:20:697] MplayerProcess::parseLine: '[ass] Error opening font: 'Arial', 0'
```

Does it work if you select mpv as multimedia engine in preferences -> general?

**Reply by KaiFrederking** [reply]
If I change to mpv, the video won't even start. I get an "mpv finished unexpectedly" message.
Here is the log, seems to be font-related, too: 
[mpv.log](https://github.com/user-attachments/files/16410393/mpv.log)

P.S.: I'm not aware of having changed anything regarding fonts recently, except for the recent MPlayer-update.

**Comment by smplayer-dev** [comment, marked as answer]
It seems you're passing the option `-nofontconfig` to mpv (preferences -> advanced -> mpv/mplayer), but that's an option for mplayer. Remove it. Maybe that option is causing the problem with the subtitles with mplayer as well.

**Reply by KaiFrederking** [reply]
Spot on! Thank you! That option was set and the second I removed it, subtitles came back on.
The only mystery remaining is, what set that option. I certainly didn't do it explicitly.
![Untitled](https://github.com/user-attachments/assets/03350c3e-5e48-4c1c-afdb-1694dfcbfab6)

**Reply by KaiFrederking** [reply]
Again, I appreciate your help at this!
![image](https://github.com/user-attachments/assets/6e8c74b9-4a16-4658-9ac7-5055c9823701)
